packages/fx-core/src/core/question.ts

```diff
@@ -345,6 +345,10 @@ export function createCapabilityQuestionPreview(inputs?: Inputs): SingleSelectQu
     );
     WorkflowOptionItem.label = `$(hubot) ${getLocalizedString("core.WorkflowOption.label.abTest")}`;
     WorkflowOptionItem.detail = getLocalizedString("core.WorkflowOption.detail.abTest");
+    DashboardOptionItem.label = `$(browser) ${getLocalizedString(
+      "core.DashboardOption.label.abTest"
+    )}`;
+    DashboardOptionItem.detail = getLocalizedString("core.DashboardOption.detail.abTest");
```

Comment:
Also update the comment?
```
// AB test for notification/command/workflow bot template naming
```

Reply:
Yes, I missed this. I've updated the comment.